cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -1985,10 +1990,21 @@ subroutine makemask
          jhi = this_block%jhi
 
          ! needs to cover halo (no halo update for logicals)
-         tmask(:,:,iblk) = .false.
-         umask(:,:,iblk) = .false.
-         nmask(:,:,iblk) = .false.
-         emask(:,:,iblk) = .false.
+         tmask(:,:,iblk)    = .false.
+         umask(:,:,iblk)    = .false.
+         umaskCD(:,:,iblk) = .false.
+         nmask(:,:,iblk)    = .false.
+         emask(:,:,iblk)    = .false.
+         do j = jlo-nghost, jhi
+         do i = ilo-nghost, ihi
+!!!!!WARNING THIS SUM is not calculating uvmCD mask on the northern and eastern halo
```

Comment:
This is fixed in #38, and you can probably just replace this ice_grid.F90 file with that one.

Reply:
> @TillRasmussen, is #39 in #33 now? Can/should we cancel #39?

Yes I think so